src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="field"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      data-qa="big"
+      method="GET"
+    >
+      <div>
+        <img
+          class="big-icon"
+          src="images/Search.svg"
+          alt="search"
+        >
+        <input
+          class="search-one"
```

Comment:
```suggestion
          class="input input-big"
```
search-one та search-two - не дуже інформативно